path/src/deno.ps1

```
@@ -0,0 +1,88 @@
+	$deno_update_channel = "stable"
+	if ($deno_ver.Contains("+")) {
+		$deno_update_channel = "canary"
+	}
+	elseif ($deno_ver.Contains("-rc")) {
+		$deno_update_channel = "rc"
+	}
+
+	$upgradedFlag = Join-Path $FOUNT_DIR 'data/installer/deno_upgraded'
+	if (Test-Path $upgradedFlag) {
+		Start-Job -ScriptBlock { # 因为需要兼容 windows powershell，所以不能像是sh中一样定义和复用base_deno_update
+			param($deno_update_channel)
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'deno_update_channel' has been declared but not used.

(PSReviewUnusedParameter)